lib/uklibid/include/uk/libid.h

```diff
@@ -51,6 +51,25 @@ extern "C" {
 #define uk_libid_count() \
 	(__UKLIBID_COUNT__)
 
+/**
+ * Return the library name for a given identifier
+ *
+ * @param libid Library identifier
+ * @return Reference to a '\0'-terminated C-string that contains the library
+ *         name, a __NULL reference if the library identifier could not be
+ *         resolved
+ */
+const char *uk_libname(unsigned int libid);
+
+/**
+ * Return the library name of the current library compilation
+ *
+ * @return Reference to a '\0'-terminated C-string that contains the library
+ *         name
+ */
+#define uk_libname_self(libid) \
```

Comment:
Looks like the `libid` parameter creeped in here.

Reply:
It did... nasty little parameter...